src/hearings/components/listing-information-summary/listing-information-summary.component.html

```diff
@@ -28,12 +28,6 @@ <h2 class="govuk-heading-m">Listing information summary</h2>
 
 <ng-template #listingContent>
   <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
-    <div class="govuk-summary-list__row" *ngIf="!isCaseStatusListed()">
-      <dt class="govuk-summary-list__key">Status</dt>
-      <dd class="govuk-summary-list__value" [innerHtml]="answerSource.HEARING_RESPONSE_STATUS | transformAnswer:hearingState$ | async">
-      </dd>
-      <dd class="govuk-summary-list__actions"></dd>
-    </div>
```

Comment:
is this required to remove this status?

Reply:
https://tools.hmcts.net/jira/browse/EUI-5357 doesn't mention 'status' in listing summary' and in screenshot

https://tools.hmcts.net/jira/browse/EUI-6104